### ISSUE TITLE
Route change doesn't fire on every route?

### ISSUE THREAD

**Issue by chanmathew** [dansmaculotte/vue-segment#1]
Hey everyone,

First of all thanks for making this wrapper.

Just had a question regarding passing in the `router` property into the Vue.use function, I've passed it in as such:
```
Vue.use(Segment, {
  writeKey: process.env.VUE_APP_SEGMENT_API_KEY,
  disabled: false,
  router,
  settings: {}
})
```

However I notice that only hard page loads (refreshing the page or entering the URL in browser) will trigger an event for route change, but not when clicking through the SPA. 

Is this the expected behavior? Because when I add an `afterEach` call manually in my router config, it fires on every route change, which I thought is the intended behavior?

**Comment by Zevran** [comment]
Hi @chanmathew !

Thanks for reporting this issue, indeed you are not supposed to manually implement afterEach method to get it working.

I don't have so much time actually but I will try to resolve it as soon as I can !

Regards

**Comment by Zevran** [comment]
The fix is released with [v0.2.7](https://github.com/dansmaculotte/vue-segment/releases/tag/v0.2.7)

**Comment by chanmathew** [comment]
@Zevran Just tested and it's working, thanks so much!
